charts/bitbucket-runner-autoscaler/values.yaml

```diff
@@ -124,7 +124,6 @@ runner:
             topologySpreadConstraints: {{ .Values.runner.topologySpreadConstraints | toYaml | nindent 16 }}
             affinity: {{ .Values.runner.affinity | toYaml | nindent 16 }}
             priorityClassName: {{ .Values.runner.priorityClassName }}
-            serviceAccountName: {{ include "bitbucketRunnerAutoscaler.runnerServiceAccountName" . }}
```

Comment:
This will run the runner using which ServiceAccount? I am assuming the default on in the Namespace. If so, it is not a best practice and we should offer a way how to run it using a different SA anyway.